src/Driver.cpp

```diff
@@ -194,6 +194,18 @@ void Driver::openURI(std::string const& uri)
     { // UDP udp://hostname:remoteport
         if (marker == string::npos)
             throw std::runtime_error("missing port specification in udp:// URI");
+
+        string::size_type second_marker = device.find_last_of(":");
+        int extended_info = 0;
+
+        if (second_marker != string::npos)
+        {
+            extended_info = boost::lexical_cast<int>(device.substr(second_marker + 1));
```

Comment:
The reason why `additional_info` is called like that is because it is shared across all the URIs we can parse (so, it's difficult to give it a better name). In this case, `extended_info` is really the local port number, name it accordingly